cdap-docs/application-templates/source/etl/templates/transformations/projection.rst

```diff
@@ -6,19 +6,35 @@
 Transformations: Projection
 ===========================
 
-.. rubric:: Description: Projection Transform: Drop, Rename, and Cast Fields
+.. rubric:: Description
 
-Projection transform that lets you drop, rename, and cast fields to a different type.
+A transform that can drop fields, rename fields, and cast fields to a different type.
+Operations are performed in a single step and are not applied in order on top of each other.
+For example, suppose the transform is configured to drop field 'B' and rename field 'A' to 'B'.
+If the transform receives an input record with field 'A' assigned a value of 10, and field 'B'
+assigned a value of 20, input field 'B' will be dropped and input field 'A' will be renamed
+to 'B'. Thus the output record will contain a single field named 'B' assigned a value of 10.  
```

Comment:
The output record will then contain a single field named 'B' with an assigned value of 10.